python-sdk/tests/test_identity.py

```diff
@@ -0,0 +1,39 @@
+from tests.fixtures import *
+
+from pachyderm_sdk.api import identity
+from pachyderm_sdk.constants import AUTH_TOKEN_ENV
+
+
+@pytest.mark.skipif(
+        not os.environ.get(AUTH_TOKEN_ENV),
+        reason="auth code not available",
+    )
+class TestIdentity:
+
+    @staticmethod
+    def test_identity_server_config(auth_client: TestClient):
+        isc = auth_client.identity.get_identity_server_config()
+        assert isc.config.issuer == "http://pachd:30658/dex"
+
+    @staticmethod
+    def test_oidc_client(auth_client: TestClient):
+        auth_client.identity.delete_all()
```

Comment:
Do we need this delete_all()? In general we want these tests to not have side effects